roseau/load_flow/models/buses.py

```diff
@@ -18,10 +18,13 @@
 class Bus(Element, JsonMixin):
     """An electrical bus."""
 
+    allowed_phases = frozenset({"ab", "bc", "ca", "an", "bn", "cn", "abn", "bcn", "can", "abc", "abcn"})
+
     def __init__(
         self,
         id: Any,
-        n: int,
+        *,
+        phases: str,
         geometry: Optional[Point] = None,
         potentials: Optional[Sequence[complex]] = None,
         ground: Optional[Ground] = None,
```

Comment:
I think that passing the ground to a bus can be removed. Maybe when refactoring the ground object. I let you organize the transition as you prefer.